pages/supporters.html

```diff
@@ -0,0 +1,56 @@
+---
+title: "LiveWires Supporters"
+---
+<div id=hero>
+
+  <a href="/" id="logo"><span>LiveWires</span></a>
+
+</div>
+<div class=heading>
+
+  <h1>Supporters</h1>
+
+</div>
+<div class="summary">
+
+  <p>Get prayer updates for LiveWires.</p>
+
+</div>
+<form class="row row-form" method="post" action="https://livewires.us18.list-manage.com/subscribe/post?u=c2fd80738edbb1442a9d61444&id=ae92635e55">
+  <fieldset>
+
+    <label for="email"><strong>Your email:</strong> <span class="field-hint">(required)</span></label>
+    <input type="email" id="email" name="EMAIL" size="30" placeholder="bob@example.com" required>
+
+  </fieldset>
+  <fieldset class="fieldset-left">
+
+    <label for="firstname"><strong>First name:</strong> <span class="field-hint">(required)</label>
+    <input type="text" id="firstname" name="FNAME" size="18" placeholder="Bob" required>
+
+  </fieldset>
+  <fieldset class="fieldset-right">
+
+    <label for="lastname"><strong>Last name:</strong> <span class="field-hint">(required)</label>
+    <input type="text" id="lastname" name="LNAME" size="18" placeholder="Smith" required>
+
+  </fieldset>
+  <fieldset>
+
+    <legend><strong>What email format do you prefer?</strong></legend>
```

Comment:
I figure we might have some people who care about this.

Reply:
It is LiveWires…